wsgi/odyssy/events/templates/events/event_list.html

```diff
@@ -0,0 +1,17 @@
+<body>
+    {% for event in events %}
+   <li><a href="{% url 'event-view-single' event.id %}">{{ event }}</a></li>
+    {% endfor %}
+
+    <div id="events-list-pagination">
+        <span class="step-links">
```

Comment:
Sorry sir, I was using original website to test. CSS file was included at that time. When I was creating new template, I just did copy paste. Anyways, after some time, this file is going to be change by someone, hence changing this file now is irrelevant. So, shall I change this?